src/rt/region/region_rc.h

```diff
@@ -224,6 +224,7 @@ namespace verona::rt
     static void gc_cycles(Alloc& alloc, Object* o, RegionRc* reg)
     {
       assert(o->get_class() == RegionMD::OPEN_ISO);
+      (void)o;
```

Comment:
I have to complain on such a small thing.  But we try to write 
```
UNUSED(o);
```
to make it more clear.